src/transformState.js

```diff
@@ -61,8 +61,28 @@
  * @param {Object} state
  * @param {Object[]} transforms
  */
+
 function transformState(state, transforms) {
-  // write code here
+  for (const transform of transforms) {
+    switch (transform.operation) {
+      case 'addProperties':
+        Object.assign(state, transform.properties);
+        break;
+      case 'removeProperties':
+        for (const value of transform.properties) {
+          delete state[value];
+        }
```

Comment:
it's not value, it's key
```suggestion
        for (const key of transform.properties) {
          delete state[key];
        }
```